packages/database/scripts/dumpDatabase.sh

```diff
@@ -29,7 +29,8 @@ function show_loading_spinner() {
     echo "" # reset prompt
 }
 
-source ".env"
+DIR=$(pwd "$0")
+source $DIR/../../scripts/bash/mergeEnvForDB.sh 
```

Comment:
Does this safely handle directories with special characters (like spaces)? (I’m not enough of a shell wizard to know)

If not:

```suggestion
source "$DIR/../../scripts/bash/mergeEnvForDB.sh"
```

I think there are a few instances of this throughout this PR